lib/rules/no-restricted-imports.js

```diff
@@ -104,9 +107,9 @@ const arrayOfStringsOrObjectPatterns = {
                     }
                 },
                 additionalProperties: false,
-                required: ["group"],
                 not: {
                     anyOf: [
+                        { required: ["group", "regexGroup"] },
```

Comment:
After this change, it seems that it would be valid to omit both properties?

Reply:
Yeah, is it possible to require either one of them?

Reply:
I think we could enforce this by `oneOf`:

```js
not: {
    anyOf: [
        { required: ["importNames", "allowImportNames"] },
        { required: ["importNamePattern", "allowImportNamePattern"] },
        { required: ["importNames", "allowImportNamePattern"] },
        { required: ["importNamePattern", "allowImportNames"] },
        { required: ["allowImportNames", "allowImportNamePattern"] }
    ]
},
oneOf: [
    { required: ["group"] },
    { required: ["regexGroup"] }
]
```

Reply:
Yes, updated.